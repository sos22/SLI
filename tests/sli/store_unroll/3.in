depth = 5
A true
B true
A -> B
B -> A
Roots:
A
End of roots
