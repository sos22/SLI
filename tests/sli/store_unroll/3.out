A true
B true
A2 dupe
B2 dupe
A3 dupe
B3 dupe
A -> B
B -> A2
A2 -> B2
B2 -> A3
A3 -> B3
Roots:
A
End of roots
